### ISSUE TITLE
No need for virtual webcam output anymore ?

### ISSUE THREAD

**Issue by olberger** [jpetazzo/obs-docker#1]
AFAIU, the stock OBS in Linux testing already ships with the V4L virtual webcam output, so I guess this might no longer be needed here.

Hope this helps,

**Comment by jpetazzo** [comment]
Yes! This is a relatively recent feature (I noticed it a few weeks ago when doing a routing upgrade on my streaming machine). I didn't see if there was any way to configure it, however (it seems to always pick the first V4L device).

I might leave it here for now, for folks stuck on Debian stable or Ubuntu LTS, perhaps.
